lib/internal/worker.js

```diff
@@ -185,6 +185,8 @@ class Worker extends EventEmitter {
     this[kParentSideStdio] = { stdin, stdout, stderr };
 
     const { port1, port2 } = new MessageChannel();
+    const transferList = [port2];
+    if (options.transferList) transferList.push(...options.transferList);
```

Comment:
```suggestion
    if (ArrayIsArray(options.transferList)) transferList.push(...options.transferList);
```

Reply:
Improved! Thank you!!!

Reply:
@juanarbol @Himself65 Hate to be late to this, but can we please undo this? I don’t see any reason why we wouldn’t accept any iterable here. We’re also quite flexible for the `transferList` argument for `.postMessage()` in accordance with the web spec.

Reply:
No problem!